jupyter-docker/jupyter_notebook_config.py

```diff
@@ -10,4 +10,9 @@
 c.NotebookApp.ip = '*'
 c.NotebookApp.port = 8000
 c.NotebookApp.open_browser = False
-c.NotebookApp.token = ''
\ No newline at end of file
+c.NotebookApp.token = ''
+c.NotebookApp.alloworigin
+
+fragment = os.environ['GOOGLE_PROJECT'] + '/' + os.environ['CLUSTER_NAME'] + '/'
```

Comment:
There's a typo here, my version in https://github.com/broadinstitute/leonardo/pull/21 is slightly different:
```
fragment = os.environ['GOOGLE_PROJECT'] + '/' + os.environ['CLUSTER_NAME']
c.NotebookApp.base_url = '/api/notebooks/' + fragment + '/'
c.NotebookApp.webapp_settings = {'static_url_prefix':'/api/notebooks/' + fragment + '/static/'} 
```